acados/sim/sim_common.c

```diff
@@ -133,15 +133,15 @@ int sim_out_calculate_size(void *config_, void *dims)
     int NF = nx + nu;
     size += sizeof(sim_info);
 
-    size += nx * sizeof(double);                   // xn
-    size += nx * NF * sizeof(double);              // S_forw
-    size += (nx + nu) * sizeof(double);            // S_adj
-    size += ((NF + 1) * NF / 2) * sizeof(double);  // S_hess
+    size += nx * sizeof(double);                // xn
+    size += nx * NF * sizeof(double);           // S_forw
+    size += (nx + nu) * sizeof(double);         // S_adj
+    size += (NF * NF) * sizeof(double);         // S_hess
```

Comment:
Is this the answer to the above comment? The way data is stored in S_hess has changed?